src/QsCompiler/Tests.Compiler/TestCases/StringParsingTests/StringParsing.qs

```diff
@@ -32,6 +32,14 @@ namespace Microsoft.Quantum.Testing.LocalVerification {
         let str = "//";
     }
 
+    operation NotebookStringParsingTest1 () : Unit {
+        let str = "hi"; %
+    }
+
+    operation NotebookStringParsingTest2 () : Unit {
+%simulate SampleRandomNumber nQubits=3
+    }
```

Comment:
These two test cases seem like they should work as-is, but they are also pretty tied to the magic command parsing that is delayed now. Would it be worth pulling them out for now in case the nature or strategy of the test changes once the feature is pulled back in?

Reply:
Thanks, I'll nuke this for now